app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java

```diff
@@ -522,6 +522,17 @@ public static void reportLoginClicks() {
         reportEvent(CCAnalyticsEvent.LOGIN_CLICK);
     }
 
+    public static void continueButtonClicks() {
+        reportEvent(CCAnalyticsEvent.CONTINUE_BUTTON_CLICKED);
+    }
+    public static void sentMessage() {
+        reportEvent(CCAnalyticsEvent.SENT_MESSAGE);
+    }
+
+    public static void personalIdLinked() {
```

Comment:
- we should send the app id we are linking here as a custom parameter.
- Think we should also log if user denies the linking or linking fails due to any reason as part of this event